src/views/Terms/TermList.jsx

```diff
@@ -31,7 +31,13 @@ const TermList = ({ matchType, query, type }) => {
 	const [
 		{ language, canonicalHost, dictionaryTitle, siteName, basePath },
 	] = useStateValue();
-	const { DefinitionPath, HomePath } = useAppPaths();
+	const {
+		DefinitionPath,
+		HomePath,
+		ExpandPath,
+		ExpandPathSpanish,
+		SearchPath,
+		SearchPathSpanish } = useAppPaths();
```

Comment:
Did you run the linter on this? I would expect that `} = useAppPaths();` would get placed on a new line

Reply:
I used the short cut `option + shift + F`

Reply:
moved to a new line